Core/Systems/EntityModifier.cs

```diff
@@ -76,9 +80,14 @@ public void ApplyTo(Player player)
 		ps.MaxMana = (int)MaxHealthPotions.ApplyTo(ps.MaxMana);
 		ps.ManaPower = (int)PotionManaPower.ApplyTo(ps.ManaPower);
 		ps.ManaDelay = (int)PotionManaDelay.ApplyTo(ps.ManaDelay);
+
+		BuffModifierPlayer buffPlayer = player.GetModPlayer<BuffModifierPlayer>();
```

Comment:
Should these be inside [here](https://github.com/Path-of-Terraria/PathOfTerraria/blob/main/Core/Systems/ModPlayers/UniversalBuffingPlayer.cs) instead?

Reply:
I don't think so, this matches the pre-existing format in ApplyTo and would be easier to find.